qutebrowser/browser/browsertab.py

```diff
@@ -650,6 +650,7 @@ def __init__(self, *, win_id, mode_manager, private, parent=None):
         # self.action = AbstractAction()
 
         self.data = TabData()
+        self._input_mode = usertypes.KeyMode.normal
```

Comment:
This is a private attribute, so you shouldn't read/write it from the outside. This should probably go into the `TabData` object instead.